packages/frontend/src/components/match_card.tsx

```diff
@@ -0,0 +1,36 @@
+export function MatchCard(props: { score: Record<string, number> }) {
```

Comment:
Could we change the name to `VersusMatchCard` (along with the filename) so it doesn't collide with 2 other match cards